OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -163,13 +168,10 @@ public MinefieldOrderGenerator(Actor a, CPos xy, bool queued)
 				minefieldStart = xy;
 				this.queued = queued;
 
-				var tileset = a.World.Map.Tileset.ToLowerInvariant();
-				if (a.World.Map.Rules.Sequences.HasSequence("overlay", "build-valid-{0}".F(tileset)))
```

Comment:
Same here.
We could still use `"{0}-{1}".F(minelayer.Info.TileValidName, tileset)` if such a sequence exists.

Reply:
Good idea. Done.